### DISCUSSION TITLE
Coloured icons for signal strength sensors

### DISCUSSION THREAD

**Discussion by MMK21Hub** [home-assistant/frontend#14683] [Dashboard]
With the 2022.12 update, my Glance cards look great thanks to the addition of colours that represent states. It would be great if sensors for Bluetooth/WiFI signal strength were also coloured red/yellow/green based on their state. The `signal_strength` device class already exists to facilitate this.

Since only the Mobile App integration currently displays a different icon based on the signal strength value, it would also be nice if the frontend did this automatically for signal strength sensors.

The `battery` device class already has both of these features implemented (icon and colour based on state).

![Screen Shot 2022-12-10 at 07 15 27](https://user-images.githubusercontent.com/50421330/206837665-24ff4035-ff47-4705-a474-26ff027197a4.png)

![Screen Shot 2022-12-10 at 08 07 47](https://user-images.githubusercontent.com/50421330/206840319-d974d536-aa91-46d1-aa98-38be5ebe7f49.png)

**Comment by ChristophCaina** [comment]
in this case, I do have some more options :)

Signal Strenght (all entities that provide "dBM" as unit of measurement)
![grafik](https://user-images.githubusercontent.com/26391061/206842747-fe0ad436-1b0a-4a04-b1da-1f89dd2314a8.png)

And maybe also to consider other entities that provide a percentage as Unit of measurement.
But in this case, It would be great if it could be configured if 0% is good or 100% :)

Low percentage: Good -> green
![grafik](https://user-images.githubusercontent.com/26391061/206842785-46db4e64-a41f-4934-97d7-83e7c3bb0708.png)

in this case, like batteries: high percentage => green 
![grafik](https://user-images.githubusercontent.com/26391061/206842916-83b16d80-fba7-4a15-97c1-456a74888f25.png)
